packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import ExpandCollapsePanel, { PanelTrigger } from '../ExpandCollapsePanel';
+import randomId from '../../utils/rndid';
+import { IconType } from '../Icon';
+
+interface AccordionProps extends React.HTMLAttributes<HTMLDivElement> {
+  className?: string;
+  children: React.ReactNode | React.ReactNode[];
+  props?: React.HTMLAttributes<HTMLDivElement>;
+}
+
+const Accordion = ({ className, children, ...props }: AccordionProps) => {
+  return (
+    <div
+      className={classNames(className ? className : '', 'Accordion__container')}
+      {...props}
+    >
+      {children}
+    </div>
+  );
+};
+
+interface AccordionContainerProps extends React.HTMLAttributes<HTMLDivElement> {
+  className?: string;
+  children: React.ReactElement[];
+  open?: boolean;
+  setIsOpen?: (open: boolean) => void;
+  trigger?: React.ReactNode | React.ReactNode[];
+  iconExpanded?: IconType;
+  iconCollapsed?: IconType;
+  shouldHideIcon?: boolean;
```

Comment:
```suggestion
  hideIcon?: boolean;
```

The above reads a bit cleaner and is more consistent with how we name our props elsewhere. Additionally this should likely be on the trigger.